src/components/common/SearchPannel/SearchPannel.jsx

```diff
@@ -0,0 +1,19 @@
+import './searchPannel.scss';
+
+const SearchPannel = () => {
+  return (
+    <form className="search-container" action="/search" method="GET">
+      <button type="submit" className="search-button">
```

Comment:
Usually, if button contains only an icon without a label, it should have `aria-label` attribute describing the action that button do